src/ops.rs

```diff
@@ -74,7 +74,7 @@ pub enum Angle45 {
     Last = 8,
 }
 
-#[derive(Copy, Clone, Debug, FromPrimitive)]
+#[derive(Copy, Clone, Debug, FromPrimitive, ToPrimitive)]
```

Comment:
This has to be changed in the generator instead. This file is generated and although it could be generated in build time, it was causing too many issues due to the need of compile C code for it. I decided to manually generate it to avoid this problem.

Reply:
Oh, sorry; I didn't realise that. So I guess I should 1) add this to generator/build.rs line 807, and 2) run `build.sh`?

Do you mind if I add a note about the generator setup in the README to help future contributors?

Reply:
Go for it. 
And yes about 1 and 2